k2/csrc/determinize.cc

```diff
@@ -28,6 +28,7 @@ LogSumTracebackLink::LogSumTracebackLink(
 
 int32_t GetMostRecentCommonAncestor(
     std::unordered_set<LogSumTracebackState *> *cur_states) {
+  if (cur_states->size() == 1) return 1;
```

Comment:
My intention was that it return 0 in this case; even the documentation says that.  What happened if it returned zero?

Reply:
if it's `0`, there `label` will not be used in `DetStateToCompact`
https://github.com/danpovey/k2/blob/75c4cd1b1fdd2a20009997c269b905b438575cdb/k2/csrc/determinize.h#L745-L746

Different states with same `seq_len` will mapped to a same `det_state`.